game.js

```diff
@@ -0,0 +1,90 @@
+const Frame = require("./frame");
+
+class Game {
+  constructor() {
+    this.frames = [];
+    this.lastFrame = 10;
+    this.bonusFrame = 11;
+  }
+
+  addPointsScored = (num) => {
+    this.continueGame();
+    this.addFrame();
+    this.frames[this.frames.length - 1].knocked_down_pins(num);
+    this.addBonusRolls();
+  };
+
+  continueGame = () => {
+    if (this.frames.length === this.lastFrame) {
+      if (
+        this.frames[this.frames.length - 1].pinsLog.length === 2 &&
+        this.frames[this.frames.length - 1].addScore() < 10
+      ) {
+        throw "The game has finished!";
+      }
+    }
+
+    if (
+      this.frames.length <= this.lastFrame ||
+      (this.frames.length == this.bonusFrame &&
+        this.frames[this.frames.length - 1].isAStrike()) ||
+      (this.frames.length == this.bonusFrame &&
+        this.frames[this.frames.length - 1].isASpare())
+    ) {
+      return;
+    } else throw "The game has finished!";
+  };
```

Comment:
can make methods like these private since they are not being tested and are only used inside other methods

Reply:
Absolutely agree, thank you!